README.md

```diff
@@ -74,24 +76,33 @@ If you do not wish to run from an IDE, use `gradlew.bat runClient` to run KAMI.
 
 ### Building
 #### Windows
-You can build by running `autobuild.bat` and then `autocopy.bat`, or if you prefer the manual way, do the following:
-
+You can build by running these commands (without these <>) in a terminal with the current directory being KAMI. (EG. `cd C:\Users\Username\Downloads\KAMI`)
 ```
-gradlew.bat build
-cd build/libs
+gradlew.bat <args>
 ```
-To copy the release  jar in `build/libs` you can run `autocopy.bat` or, if you prefer doing it manually, find a file `KAMI-<minecraftVersion>-<kamiVersion>-release.jar` which you can copy to the `mods` folder of a minecraft instance that has forge installed.
+Possible arguments on Windows are `build`.
+
+To copy on windows run `autocopy.bat`
+
+If you prefer copying it manually, find a file in `build/libs` called `KAMI-<minecraftVersion>-<kamiVersion>-**release**.jar` which you can copy to the `mods\1.12.2` folder of a minecraft instance that has forge installed.
+
+Note: This assumes your minecraft folder is in the default location under your home folder.
+
 #### Linux
+You can build by running these commands (without these <>) in a terminal with the current directory being KAMI. (EG. `cd ~/Downloads/KAMI`)
 ```
-git clone https://github.com/zeroeightysix/KAMI.git
-cd KAMI
-chmod +x auto*
-./autobuild
-mkdir ~/.minecraft/mods
-./autocopy
+./gradlew <args>
 ```
+Possible arguments are `build`, `mkdir`, `rmOld` and `copy`, in that order. 
+
+Build is required, mkdir makes the mods/1.12.2 directory, rmOld removes old versions of KAMI in that directory, and copy copies the build release to the mods/1.12.2 directory. 
```

Comment:
Reformat the names to be in `code blocks`?

Reply:
Updated